src/Service/SearchService.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Service;
+
+use Doctrine\ORM\QueryBuilder;
+
+class SearchService
+{
+    public function getFieldMatches(string $search): array
```

Comment:
It's not obvious what this function actually does. Please add a descriptive comment.

What happens if I want to search for a phone number (with 8 digits)?

Reply:
Added comment to describe functions in `SearchService`.

Adding a search for phone number as mentioned would be done by adding another `preg_match` in `getFieldMatches ` and a `orWhere` in `applyFieldSearch`. As a consequence we would just find cases where the search matches a telephone or a case number.